boards/nrf51dk/src/main.rs

```diff
@@ -261,15 +261,35 @@ pub unsafe fn reset_handler() {
 
 
 use core::fmt::Arguments;
+use core::fmt::Write;
+use core::fmt::write;
```

Comment:
Since Branden's already got you making changes, we've been doing `use core::fmt::{Write, write}` instead of the two-liner elsewhere. We're not 100% consistent (mostly b/c rustfmt doesn't enforce this yet), but let's at least try :)